kyaml/yaml/fns.go

```diff
@@ -541,6 +541,9 @@ func (l PathGetter) getFilter(part, nextPart string, fieldPath *[]string) (Filte
 	case part == "-":
 		// part is a hyphen
 		return GetElementByIndex(-1), nil
+	case part == "*":
+		// part is a asterisk
+		return nil, errors.Errorf("cannot support create option in a multi-value target now")
```

Comment:
Won't it be possible for someone to be attempting to use `PathGetter` with an asterisk but without the create option? I think this error message needs to be about wildcard support rather than create mode specifically because of that (and replacements may need to augment the error to clarify what the problem is in that specific flow).

Reply:
I think I fix it to https://github.com/kubernetes-sigs/kustomize/pull/4577/commits/01ab069bd2a74f29a5e02fca3a9d8ff79465a145